MessengerOnFileForLinux/CMakeLists.txt

```diff
@@ -7,17 +7,15 @@ set(CMAKE_CXX_EXTENSIONS ON)
 set(LIBRARY_OUTPUT_PATH build/)
 set(BINARY_OUTPUT_PATH build/)
 set(EXECUTABLE_OUTPUT_PATH ${PROJECT_BINARY_DIR}/bin)
+set(CMAKE_CXX_FLAGS "--coverage")
 
 set(BUILD_SHARED_LIBS OFF)
-# OUTSIDE LIBRARIES
+include(External_GTest.cmake)
 include(CTest)
```

Comment:
czy to jest potrzebne w związku z External_Gtest.cmake?

Reply:
sprawdze, jak nie to wywalam